source/connecting/onprem-to-atlas.txt

```diff
@@ -21,6 +21,10 @@ Atlas cluster using {+c2c-full-product-name+}.
 For additional details on connecting to an Atlas cluster, see
 :ref:`Connect to a Database Deployment <atlas-connect-to-deployment>`
 
+.. note::
+
+   .. include:: /includes/migration-upgrade-recommendation.rst
```

Comment:
For on prem to atlas, the atlas cluster will be updated by mongodb in a rolling manner, so they only need to worry about the on-prem cluster. I think we should make that clear.

Reply:
Done!

Reply:
The atlas cluster can either be the source or destination. I think that we should specify that the on prem cluster is the one that needs to be updated.

Reply:
Fixed!